pom.xml

```diff
@@ -267,7 +267,7 @@
         <formatter-maven-plugin.version>0.5.2</formatter-maven-plugin.version>
         <com.google.code.maven-license-plugin.version>1.4.0</com.google.code.maven-license-plugin.version>
         <!-- == Dependency Versions == -->
-        <postgresql.version>9.4-1206-jdbc41</postgresql.version>
+        <postgresql.version>9.4.1212</postgresql.version>
```

Comment:
- [ ] Sorry, please use not `9.4.1212` but `9.4.1212.jre7`.

Because we must test with JDK 7. `9.4.1212` with JDK 7 cause `java.lang.UnsupportedClassVersionError`.